packages/terra-pills/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-pills",
```

Comment:
It'd be more consistent to name this package after one of the components inside it (terra-pill-list/terra-pill)

Though to be honest, I wouldn't be opposed to keeping the package name and renaming PillList to just Pills either. Not sure the history there, but it's not a semantic list, doesn't have a role of list, doesn't feature item counts or anything. It's more a collection than anything.

Reply:
CC : @neilpfeiffer

Reply:
I'm wondering this too. The terminology seems awkward. Also since we want to heavily promote the use of the 'pilllist' to contain pills as it's the default export. We could make pill a subcomponent of pills to further tie the relationship.

```jsx
import React from 'react';
import Pills from 'terra-pills';

const BasicPill = () => (
  <Pills
    ariaLabel="Example of a Basic Pill"
  >
    <Pills.Pill
      label="asthma"
    />
  </Pills>
);

export default BasicPill;
```

or

```jsx
import React from 'react';
import PillsContainer from 'terra-pills';

const BasicPill = () => (
  <PillsContainer
    ariaLabel="Example of a Basic Pill"
  >
    <PillsContainer.Pill
      label="asthma"
    />
  </PillsContainer>
);

export default BasicPill;
```